python/lsst/meas/base/forcedPhotCcd.py

```diff
@@ -143,7 +143,7 @@ class ForcedPhotCcdConfig(ForcedPhotImageConfig):
     doApplyUberCal = lsst.pex.config.Field(
         dtype=bool,
         doc="Apply meas_mosaic ubercal results to input calexps?",
-        default=False
+        default=True
```

Comment:
1. doApplyUberCal=False should definitely be the default for the whole stack.
1. I don't think we should call it ubercal, if it's not ubercal (which it isn't).
1. meas_mosiac only works on HSC. jointcal works on cfht and hsc (and sort of on lsstSim), and we're currently working on making it work on general camera data (metadata handling is the problem).